soc/arm64/renesas_rcar/gen3/mmu_regions.c

```diff
@@ -37,9 +39,12 @@ static const struct arm_mmu_region mmu_regions[] = {
 			      MT_DEVICE_nGnRnE | MT_RW | MT_NS),
 #endif
 
+#if DT_NODE_HAS_STATUS(DT_INST(0, renesas_rcar_pfc), okay)
 	MMU_REGION_FLAT_ENTRY("PFC",
-			      DT_REG_ADDR(DT_INST(0, renesas_rcar_pfc)), 4096,
```

Comment:
Please, make your commit consistent, you may change everything before merging. Add proper changes to dedicated commits.

Reply:
done